tests/cli/lib/lib.sh

```diff
@@ -132,7 +132,7 @@ composer_stop() {
 verify_image() {
     SSH_USER="$1"
     SSH_MACHINE="$2"
-    SSH_OPTS="-o StrictHostKeyChecking=no $3"
+    SSH_OPTS="-o StrictHostKeyChecking=no -o BatchMode=yes $3"
```

Comment:
Also add to `tests/cli/test_build_and_deploy_azure.sh` for cloud-init checks.

Reply:
Added. Does it actually do anything if the identity file is specified as well?

Reply:
I think the only difference is when you run the commands on interactive terminal and for commands which are supposed not to be able to authenticate.